### PR TITLE
ignore the year which has no running data when drawing github.svg

### PR DESCRIPTION
**Pull request by Narglc**
the last github.svg like:
image

### CODE REVIEW

File: run_page/gpxtrackposter/year_range.py [hunk 1]
```diff
@@ -82,5 +88,9 @@ def count(self) -> Optional[int]:
             return None
         return 1 + self.to_year - self.from_year
 
+    def count_with_data(self):
+        """Return number of years which has running data"""
+        return len(self.years_dict)
+
```
Comment (yihong0618) [inline run_page/gpxtrackposter/year_range.py:94]: can we just make it a property?

like

@property
def real_year():
    pass

File: run_page/gpxtrackposter/github_drawer.py [hunk 2]
```diff
@@ -37,6 +37,9 @@ def draw(self, dr: svgwrite.Drawing, size: XY, offset: XY):
             )
             year_length = total_length_year_dict.get(year, 0)
             year_length = format_float(self.poster.m2u(year_length))
+
+            if year_length == "0.0":
```
Comment (yihong0618) [inline run_page/gpxtrackposter/github_drawer.py:41]: if str(year_length)

File: run_page/gpxtrackposter/year_range.py (hunk 1 above)
Comment (Narglc) [reply run_page/gpxtrackposter/year_range.py]: got it!

**Review by yihong0618 — APPROVED** [review]
LGTM

**Comment by yihong0618** [comment]
Thanks

**Pull request merged**